package.json

```diff
@@ -14,24 +14,20 @@
   },
   "main": "index.js",
   "scripts": {
-    "start": "NODE_ENV=production node --trace-warnings ./src/server/server.babel.js",
+    "start": "cross-env NODE_ENV=production node ./src/server/server.babel.js",
```

Comment:
Had to read the docs for `cross-env`. Elegant. I hope that helps our Windows friends.